theano/sandbox/rng_mrg.py

```diff
@@ -285,11 +286,17 @@ def mrg_next_value(rstate, new_rstate):
 class mrg_uniform_base(Op):
     # TODO : need description for class, parameter
     __props__ = ("output_type", "inplace")
+    params_type = ParamsType(inplace=bool_t, ndim=int_t, otypenum=int_t, otype_is_float32=bool_t)
```

Comment:
In this case, `ndim`, `otypenum` and `otype_is_float32` are got from `self.output_type` which is indeed used to create an output type of the Op. But we can not be sure than output storages are allocated or even have right properties when we are in C code (an output pointer may point to NULL).

So I think I should better take these properties here. I will add comments at this line to explain that ?